src/library/scala/collection/Seq.scala

```diff
@@ -553,6 +553,8 @@ trait SeqOps[+A, +CC[_], +C] extends Any
     *
     *  @return   An Iterator which traverses the possible n-element combinations of this $coll.
     *  @example  `"abbbc".combinations(2) = Iterator(ab, ac, bb, bc)`
+    *
+    *            `"bab".combinations(2) = Iterator(bb, ba)`
```

Comment:
Did you check that this renders correctly?

Reply:
Let me check it locally.

Reply:
now fixed.